rwengine/src/data/AnimGroup.cpp

```diff
@@ -1,39 +1,390 @@
+             {"car_closedoor_lhs", 0x18},
+             {"car_closedoorl_lhs", 0x18},
+             {"car_rolldoor", 0x18},
+             {"car_rolldoorlo", 0x18},
+             {"car_getout_lhs", 0x18},
+             {"car_getoutl_lhs", 0x18},
+             {"car_close_lhs", 0x18},
+             {"car_align_rhs", 0x18},
+             {"car_alignhi_rhs", 0x18},
+             {"car_open_rhs", 0x18},
+             {"car_doorlocked_rhs", 0x18},
+             {"car_pullout_rhs", 0x18},
+             {"car_pulloutl_rhs", 0x18},
+             {"car_getin_rhs", 0x18},
+             {"car_getinl_rhs", 0x18},
+             {"car_closedoor_rhs", 0x18},
+             {"car_closedoorl_rhs", 0x18},
+             {"car_shuffle_rhs", 0x18},
+             {"car_lshuffle_rhs", 0x18},
+             {"car_sit", 0x4},
+             {"car_lsit", 0x4},
+             {"car_sitp", 0x4},
+             {"car_sitplo", 0x4},
+             {"drive_l", 0x14},
+             {"drive_r", 0x14},
+             {"drive_lo_l", 0x14},
+             {"drive_lo_r", 0x14},
+             {"driveby_l", 0x14},
+             {"driveby_r", 0x14},
+             {"car_lb", 0x14},
+             {"drive_boat", 0x4},
+             {"car_getout_rhs", 0x18},
+             {"car_getoutl_rhs", 0x18},
+             {"car_close_rhs", 0x18},
+             {"car_hookertalk", 0x12},
+             {"coach_opnl", 0x18},
+             {"coach_opnr", 0x18},
+             {"coach_inl", 0x18},
+             {"coach_inr", 0x18},
+             {"coach_outl", 0x18},
+             {"train_getin", 0x10},
+             {"train_getout", 0x18},
+             {"car_crawloutrhs", 0x18},
+             {"car_crawloutrhs", 0x18},
+             {"van_openl", 0x18},
+             {"van_getinl", 0x18},
+             {"van_closel", 0x18},
+             {"van_getoutl", 0x18},
+             {"van_open", 0x18},
+             {"van_getin", 0x18},
+             {"van_close", 0x18},
+             {"van_getout", 0x18},
+             {"getup", 0x58},
+             {"getup", 0x58},
+             {"getup", 0x58},
+             {"getup_front", 0x58},
+             {"jump_launch", 0x18},
+             {"jump_glide", 0x14},
+             {"jump_land", 0x58},
+             {"fall_fall", 0x16},
+             {"fall_glide", 0x14},
+             {"fall_land", 0x58},
+             {"fall_collapse", 0x58},
+             {"ev_step", 0x58},
+             {"ev_dive", 0x850},
+             {"xpressscratch", 0x118},
+             {"roadcross", 0x12},
+             {"turn_180", 0x18},
+             {"arrestgun", 0x50},
+             {"drown", 0x10},
+             {"cpr", 0x18},
+             {"duck_down", 0x14},
+             {"duck_low", 0x14},
+             {"rblock_cshoot", 0x14},
+             {"weapon_throwu", 0x18},
+             {"handsup", 0x50},
+             {"handscower", 0x58},
+             {"fucku", 0x21c},
+             {"phone_in", 0x10},
+             {"phone_out", 0x18},
+             {"phone_talk", 0x16}}},
+    {
```

Comment:
After a bit of twiddling I noticed this is a feature of clang-format. Adding a final comma to a list initialisation causes the list to be expanded over multiple lines instead of compacted. I've made all the entries consistent now.